api/set_number_partitions.md

```diff
@@ -1,32 +1,35 @@
-## set_number_partitions() 
+## set_number_partitions() <tag type="community">Community</tag>
+
 Sets the number of partitions (slices) of a space dimension on a
 hypertable. The new partitioning only affects new chunks.
 
 ### Required Arguments
 
-|Name|Type|Description|
-|---|---|---|
-| `hypertable` | REGCLASS | Hypertable to update the number of partitions for.|
-| `number_partitions` | INTEGER | The new number of partitions for the dimension. Must be greater than 0 and less than 32,768.|
+| Name                | Type     | Description                                                                                  |
```

Comment:
Why are you adding spaces in to the tables?

Reply:
Linter - I'll revert that